### PR TITLE
Different class room and moved away from JQuery #24

### PR DESCRIPTION
**Pull request by Earthii**
Different class room are now active, live reload between all class rooms.

Missing db to fetch messages so if we refresh the brower, chat history is gone.

### CODE REVIEW

File: public/javascript/classroom.controller.js [hunk 1]
```diff
@@ -15,7 +15,46 @@ angular.module('app.classroom', ['ngRoute'])
 
     .controller('classCtrl', classCtrl);
 
-function classCtrl ($http, $routeParams){
+function classCtrl ($http, $routeParams, $rootScope, $scope){
+    /*---------------
```
Comment (jusleg) [inline public/javascript/classroom.controller.js]: This is some fancy ascii art!

File: public/app.js [hunk 2]
```diff
@@ -11,10 +11,12 @@
     config(['$locationProvider', '$routeProvider','$httpProvider',
         function($locationProvider, $routeProvider,$httpProvider) {
             $locationProvider.hashPrefix(''); // or with '!'
-
             $routeProvider.otherwise({redirectTo: ''});
-
             // $httpProvider.defaults.useXDomain = true;
             // delete $httpProvider.defaults.headers.common['X-Requested-With'];
```
Comment (jusleg) [inline public/app.js]: Should we keep this code?

File: public/app.js [hunk 3]
```diff
@@ -11,10 +11,12 @@
     config(['$locationProvider', '$routeProvider','$httpProvider',
         function($locationProvider, $routeProvider,$httpProvider) {
             $locationProvider.hashPrefix(''); // or with '!'
-
             $routeProvider.otherwise({redirectTo: ''});
-
             // $httpProvider.defaults.useXDomain = true;
```
Comment (jusleg) [inline public/app.js]: We should not keep comments or code that do not apply to the app

File: public/javascript/classroom.controller.js [hunk 4]
```diff
@@ -15,7 +15,46 @@ angular.module('app.classroom', ['ngRoute'])
...
+        $rootScope.socket.emit('userMessage', msgObj);
+        vm.m ='';
+        return false;
+    }
+
+    //when the client receives an emit (new message) from server
+    $rootScope.socket.on(vm.classId, function(msg){
+        vm.messages.push(msg)
+        $scope.$apply();
+        // $('#messages').append($('<li>').text(msg.user +' :    '+msg.message));
```
Comment (jusleg) [inline public/javascript/classroom.controller.js]: moving on from the list element!

File: public/app.js (hunk 2 above)
Comment (Earthii) [inline public/app.js]: Potentially depends on how we handle our server requests/ Api calls.

**Comment by jusleg** [comment]
#24

**Pull request merged**